Sources/Select.swift

```diff
@@ -25,10 +25,7 @@ public struct Select: Query {
     public let tables: [Table]
 
     /// The SQL WHERE clause containing the filter for rows to retrieve.
```

Comment:
I think the documentation needs to be rephrased to explain that this can be either a Filter or a String with a raw filter.